requirements.txt

```diff
@@ -11,6 +11,7 @@ pydub==0.24.1
 resampy==0.2.2
 ffmpeg-python==0.2.0
 cma==2.7.0
+pandas==0.24.2
```

Comment:
Is there a reason for using pandas 0.24 instead of pandas 1.x?

Reply:
That's just the version I already had installed. I can test with version 1.1 and see if it works as well.